edge-functions/create-react-app/src/vitals.js

```diff
@@ -0,0 +1,41 @@
+const vitalsUrl = 'https://vitals.vercel-analytics.com/v1/vitals';
+
+function getConnectionSpeed() {
+  return 'connection' in navigator &&
+    navigator['connection'] &&
+    'effectiveType' in navigator['connection']
+    ? navigator['connection']['effectiveType']
+    : '';
+}
+
+export function sendToVercelAnalytics(metric) {
+  const analyticsId = process.env.REACT_APP_VERCEL_ANALYTICS_ID;
+  if (!analyticsId) {
+    return;
+  }
```

Comment:
Should we add instructions on how to set up this?

Reply:
https://vercel.com/docs/concepts/analytics/api